src/styles/variables.scss

```diff
@@ -0,0 +1,24 @@
+// Spacing
+$spacing_xxx_small: 0.5rem;
+$spacing_xx_small: 0.75rem;
+$spacing_x_small: 1rem;
+$spacing_small: 1.25rem;
+$spacing_medium_light: 1.5rem;
+$spacing_medium: 2rem;
+$spacing_medium_plus: 2.5rem;
+$spacing_large: 3rem;
+$spacing_x_large: 4rem;
+$spacing_xx_large: 5rem;
+$spacing_xxx_large: 6rem;
```

Comment:
I feel like we could/should use less spacing variables? I think using medium light vs medium vs medium plus won't be super noticeable and could undo some of the consistency we're hoping for in the first place. maybe just an xs, sm, md, lg, xl, xxl will give us less options to choose from while giving us enough flexibility to handle all the spacing amounts

Reply:
Yeah for sure, down to cut the list down. Would you mind making a suggestion in the figma file?

Reply:
Gotchuuuu